lms/djangoapps/branding/api.py

```diff
@@ -97,10 +97,13 @@ def get_footer(is_secure=True):
         "copyright": _footer_copyright(),
         "logo_image": _footer_logo_img(is_secure),
         "social_links": _footer_social_links(),
-        "navigation_links": _footer_navigation_links(),
+        "edx_links": _footer_edx_links(),
```

Comment:
Note: this is intended to be generic to support the Open Source community, so there shouldn't be an edX-specific section.

Reply:
@andy-armstrong are you referring to the name of the method ?

Reply:
I'm referring to the key `"edx_links"` in the dict.